### DISCUSSION TITLE
Include mention or even link to dependencies in installer?

### DISCUSSION THREAD

**Discussion by Robert01101101** [GyulyVGC/sniffnet#640] [Ideas]
Hi, thank you for creating such an awesome application! I'm really enjoying it. 

As someone who skipped all reading and went straight for the download button, I'd like to suggest including instructions or even a link if possible in the installer window to clarify the need for installing dependencies (depending on the platform, I'm on Windows). I didn't realize I needed to install Npcap.

The error message that pops up otherwise when trying to start Sniffnet is not particularly helpful, especially for casual users:

> The code execution cannot proceed because wpcap.dll was not found. Reinstalling the program may fix this problem.

Since Sniffnet is trying to be accessible to a wide range of users, I think this is an area of improvement, to help more people more easily install this useful app. It feels like a low hanging fruit that is quick to implement but impactful for casual users :)

**Comment by GyulyVGC** [comment]
Hi @Robert01101101, thanks for the suggestion.
However, as you can see from point n.4 in the [installation wiki page](https://github.com/GyulyVGC/sniffnet/wiki/Install-on-Windows), there is already such feature in the Windows installer.
